lib/suspenders/app_builder.rb

```diff
@@ -366,6 +372,20 @@ def create_heroku_apps(flags)
       create_production_heroku_app(flags)
     end
 
+    def create_heroku_pipeline
+      pipeline_name = app_name.dasherize
+
+      run_heroku "pipelines:create -a #{pipeline_name}", "staging"
+      run_heroku(
+        "pipelines:add -a #{app_name.dasherize}-staging #{pipeline_name}",
+        "staging",
+      )
+      run_heroku(
+        "pipelines:add -a #{app_name.dasherize}-production #{pipeline_name}",
```

Comment:
Right. The though process was that these are the same right now, but they are not necessarily always the same, and this might change in the future.

However, it would be great to have a name that would fit all, we just couldn't come up with one.